tests/publiccloud/ipa.pm

```diff
@@ -61,7 +61,10 @@ sub run {
     assert_script_run('rm -rf ipa_results');
 
     # fail, if at least one test failed
-    die if ($ipa->{fail} > 0);
+    if ($ipa->{fail} > 0) {
```

Comment:
whatever bad happens upload cloudregister log ? can't we be more precise ? I think parse_extra_log should give more clear answer here ...

Reply:
What do you mean? That we should only upload cloudregister if a specific test of IPA fail?

Reply:
exactly . what the point of cloudregister log in case test_sles_lscpu.py or test_sles_root_pass.py or test_soft_reboot.py failing ?

Reply:
Ok got it. Was just to lazy. Then I need to read the json results file and search for specific errors.
Let me try that.